hw01_hello_otus/go.mod

```
@@ -1,3 +1,5 @@
-module github.com/fixme_my_friend/hw01_hello_otus
+module github.com/golang-corse/hw01_hello_otus
```

```suggestion
module github.com/Al-bat-ros/golang-corse/hw01_hello_otus
```

Следует указать полный путь до модуля, чтобы он был доступен для загрузки и импорта